wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/functions.php

```diff
@@ -16,26 +16,77 @@
  */
 add_filter( 'bbp_show_lead_topic', '__return_true' );
 
+/**
+ * Register a local nav menu for non-English forums, if it doesn't already exist.
+ */
+function register_local_nav_menu() {
+	if ( substr( get_locale(), 0, 2 ) === 'en' || wp_get_nav_menu_object( 'local-navigation' ) ) {
+		return;
+	}
+
+	register_nav_menu( 'local-navigation', __( 'Local Navigation', 'wporg-forums' ) );
+}
+add_action( 'after_setup_theme', 'register_local_nav_menu' );
+
 /**
  * Provide a list of local navigation menus.
  */
 function add_site_navigation_menus( $menus ) {
-	return array(
-		'forums' => array(
-			array(
-				'label' => __( 'Welcome to Support', 'wporg-forums' ),
-				'url' => '/welcome/',
+	if ( is_admin() ) {
+		return;
+	}
+
+	if ( substr( get_locale(), 0, 2 ) === 'en' ) {
+		return array(
+			'forums' => array(
+				array(
+					'label' => __( 'Welcome to Support', 'wporg-forums' ),
+					'url' => '/welcome/',
+				),
+				array(
+					'label' => __( 'Guidelines', 'wporg-forums' ),
+					'url' => '/guidelines/',
+				),
+				array(
+					'label' => __( 'Get Involved', 'wporg-forums' ),
+					'url' => 'https://make.wordpress.org/support/handbook/contributing-to-the-wordpress-forums/',
+				)
 			),
-			array(
-				'label' => __( 'Guidelines', 'wporg-forums' ),
-				'url' => '/guidelines/',
+		);
+	} else {
+		$local_nav_menu_object = wp_get_nav_menu_object( 'local-navigation' );
```

Comment:
The `wp_get_nav_menu_object` function doesn't understand locations, so you need to get the menu at this location first, then you can use it. I'm guessing this will fix the issue [you mentioned here](https://github.com/WordPress/wordpress.org/pull/250/files#r1566580291).

```suggestion
		$local_nav_menu_locations = get_nav_menu_locations();
		$local_nav_menu_object = isset( $local_nav_menu_locations['local-navigation'] ) ? wp_get_nav_menu_object( $local_nav_menu_locations['local-navigation'] ) : false;
```

Reply:
I ended up creating a test menu on es.w.org/support to test this out, since the Location wouldn't even appear in wp-admin until after a menu existed.